src/SophiApp/Localizations/IT.xaml

```diff
@@ -31,31 +31,31 @@
-    <system:String x:Key="Localization.UwpApps">UWP Apps</system:String>
+    <system:String x:Key="Localization.UwpApps">App UWP</system:String>
     <system:String x:Key="Localization.Games">Giochi</system:String>
-    <system:String x:Key="Localization.TaskScheduler">Oprazioni pianificate</system:String>
+    <system:String x:Key="Localization.TaskScheduler">Operazioni pianificate</system:String>
     <system:String x:Key="Localization.Security">Sicurezza</system:String>
     <system:String x:Key="Localization.ContextMenu">Menu contestuale</system:String>
     <system:String x:Key="Localization.Settings">Pulizia di Windows</system:String>
     <system:String x:Key="Localization.ViewApplyingException.InApplying">Mentre si applicavano le modifiche</system:String>
-    <system:String x:Key="Localization.ViewApplyingException.HasException">c'è stato un'errore.</system:String>
+    <system:String x:Key="Localization.ViewApplyingException.HasException">si è verificato un'errore.</system:String>
     <system:String x:Key="Localization.CleanupTask.NotificationTask.Title">Informazioni importanti</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.EventTitle">Esguire una pulizia dei file temporanei e degli aggiornamenti di Windows già installati?</system:String>
-    <system:String x:Key="Localization.CleanupTask.Description">Cancellazzione dei file temporanei e degli aggiornamenti di Windows già installati con l'app Pulizia disco.</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.Description">Notifiche Pop-up per ricordarti di cancellare i file di Windows inutilizzati e gli aggiornamenti già installati.</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.Run">esegui</system:String>
-    <system:String x:Key="Localization.SoftwareDistributionTask.Event">La cache di Windows update è stata eliminata.</system:String>
-    <system:String x:Key="Localization.SoftwareDistributionTask.Description">pulendo la cartella %SystemRoot%\SoftwareDistribution\Download.</system:String>
-    <system:String x:Key="Localization.ClearTempTask.Event">La cartella dei file temporanei è stata pulita.</system:String>
-    <system:String x:Key="Localization.ClearTempTask.Description">è in corso la pulizia della cartella dei file temporanei.</system:String>
+    <system:String x:Key="Localization.CleanupTask.NotificationTask.EventTitle">Vuoi eseguire una pulizia dei file temporanei e degli aggiornamenti di Windows già installati?</system:String>
+    <system:String x:Key="Localization.CleanupTask.Description">Cancellazione dei file temporanei e degli aggiornamenti di Windows già installati con l'app Pulizia disco.</system:String>
+    <system:String x:Key="Localization.CleanupTask.NotificationTask.Description">Notifiche popup per ricordarti di cancellare i file di Windows non usati e gli aggiornamenti già installati.</system:String>
```

Comment:
pop-up
Inutilizzati sounds better

Reply:
For me "pop-up" is english. Italian si "popup".
"Non utilizzati"/"Non usati" sounds better for italian then "inutilizzatI".

Reply:
![image](https://user-images.githubusercontent.com/92474350/211359098-44e85a22-12db-4d57-b0a3-1ad7074d7e2d.png)
based on Treccani they are all ok, i generally write it with the dash but it's suggettive

Reply:
For me as explained "pop-up" (as "e-mail") sound english.
I prefer the italian form "popup" (and "email").